src/libraries/pretest.proj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.Build.Traversal">
 
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
+
   <PropertyGroup>
     <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
+    <NETCoreAppFrameworkIdentifier>.NETCoreApp</NETCoreAppFrameworkIdentifier>
```

Comment:
Is there a reason why you hardcode `NETCoreAppFrameworkIdentifier` and not use `$(TargetFrameworkIdentifier)` in L86 below?

Reply:
thanks, removed hardcode